src/main/java/core/basesyntax/service/DeepCopy.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface DeepCopy {
+    List<FruitTransaction> getDeepCopy(List<FruitTransaction> list);
```

Comment:
The interface name `DeepCopy` is too generic. It would be better to name the interface to reflect what it is copying. For example, `FruitTransactionCopy` would be more descriptive if it's specifically for creating deep copies of `FruitTransaction` objects.